lib/api.js

```diff
@@ -2033,6 +2033,38 @@ API.prototype.getFiatRate = function(opts, cb) {
     if (err) return cb(err);
     return cb(null, rates);
   });
+}
+
+/**
+ * Returns subscription status.
+ * @param {Object} opts
+ * @param {String} opts.type - Device type (ios or android).
+ * @param {String} opts.token - Device token.
+ * @returns {Object} body - Status of subscription.
+ */
+API.prototype.pushNotificationsSubscribe = function(opts, cb) {
+  var url = '/v1/pushnotifications/subscriptions/';
+  this._doPostRequest(url, opts, function(err, body) {
+    if (err) return cb(err);
+    return cb(null, body);
+  });
+};
+
+/**
+ * Returns unsubscription status.
+ * @param {String} opts - Device token (unsubscribe all) or null (unsubscribe current wallet).
```

Comment:
Also, I don't think there should be a way to unsubscribe all, what does that mean? Unsubscribe other copayers?